recipes/mason/meta.yaml

```diff
@@ -5,13 +5,23 @@ package:
   version: {{ version }}
 
 source:
-  - url:  http://packages.seqan.de/mason2/mason2-{{ version }}-Mac-x86_64_sse4.zip      # [osx]
-    sha256: d478b07effe6dd2056ead6cd816fa2b196ba1e2ffb9d6d8d98e24ad968834ee9            # [osx]
-  - url: http://packages.seqan.de/mason2/mason2-{{ version }}-Linux-x86_64_sse4.tar.xz  # [linux]
-    sha256: 3879cdd71c39947d9fc731e820c9b926912e7688d070a8431c75681848eee688            # [linux]
+  - url:  http://packages.seqan.de/seqan-src/seqan-src-2.4.0.tar.gz
```

Comment:
Please update the recipe version too

Reply:
I have added a comment to the recipe to point this out.